services/transaction-statistics/shared/utils/metadata.js

```diff
@@ -15,25 +15,36 @@
  */
 const BluebirdPromise = require('bluebird');
 
-const { HTTP: { request } } = require('lisk-service-framework');
+const { HTTP: { request }, Logger } = require('lisk-service-framework');
 
 const { DB_CONSTANT } = require('./constants');
 const { requestAppRegistry } = require('./request');
 const config = require('../../config');
+const { requestConnector } = require('./request');
+
+const logger = Logger();
 
-// TODO: Resolve network from network status response once available
 const getTokenMetadataByID = async (tokenID) => {
-	const network = 'mainnet';
+	const { chainID } = await requestConnector('getNetworkStatus');
+	const network = config.networks[chainID];
```

Comment:
Please refactor according to the changes suggested within the config file. Also, please rename the variable `network` to `networkConfig`.

Reply:
applied.